core/services/relay/evm/mercury/wsrpc/pool.go

```diff
@@ -67,6 +67,7 @@ func (conn *connection) ensureStartedClient(ctx context.Context) error {
 }
 
 func (conn *connection) checkin(checkinCco *clientCheckout) {
+	conn.lggr.Debug("Checking in client", "serverURL", conn.serverURL)
```

Comment:
The Close() method on `clientCheckout` may benefit

```
func (cco *clientCheckout) Close() error {
```

Reply:
the Close() method only delegates to checkin without any additional work - prefer to keep this in this checkin method to keep Close() small